workshops/templates/workshops/instructors.html

```diff
@@ -25,6 +27,13 @@
         <td>{{ p.num_taught }}</td>
         <td>{% if p.airport %}<a href="{{ p.airport.get_absolute_url }}">{{ p.airport }}</a>{% else %}—{% endif %}</td>
         <td>{% if p.airport %}{{ p.airport.country.name }}{% else %}—{% endif %}</td>
+        <td>{% for badge in p.badges.all %}
+        {% if badge.name == 'swc-instructor' %}
+          {% bootstrap_tag 'SWC' %}
+        {% elif badge.name == 'dc-instructor' %}
+          {% bootstrap_tag 'DC' %}
+        {% endif %}
```

Comment:
There's a way, but it will only work for '*-instructor' badges. For others, a new tag (like `bootstrap_tag`) should be introduced.